### PR TITLE
FPGA build parameter for HW_LATEST

### PR DESCRIPTION
**Pull request by jlmahowa-amd**

### CODE REVIEW

File: .github/workflows/nightly-release.yml [hunk 1]
```diff
@@ -83,7 +83,7 @@ jobs:
     uses: ./.github/workflows/fpga.yml
     with:
       artifact-suffix: -fpga-realtime-etrng-nolog
-      extra-features: slow_tests
+      extra-features: slow_tests,hw-latest
```
Comment (korran) [inline .github/workflows/nightly-release.yml:86]: Can we test both with hw-1.0 and hw-latest? I don't think they've released 1.0 yet...

File: hw-model/src/output.rs [hunk 2]
```diff
@@ -79,6 +79,9 @@ impl OutputSink {
     pub fn set_now(&self, now: u64) {
         self.0.now.set(now);
     }
+    pub fn get_now(&self) -> u64 {
```
Comment (korran) [inline hw-model/src/output.rs:82]: nitpick: In idiomatic Rust, getters don't prefix get_:

https://rust-lang.github.io/api-guidelines/naming.html#getter-names-follow-rust-convention-c-getter

File: hw-model/src/model_fpga_realtime.rs [hunk 3]
```diff
@@ -446,9 +449,11 @@ impl HwModel for ModelFpgaRealtime {
     }
 
     fn output(&mut self) -> &mut crate::Output {
-        self.output
-            .sink()
-            .set_now(self.start_time.elapsed().as_millis().try_into().unwrap());
+        self.output.sink().set_now(
```
Comment (korran) [inline hw-model/src/model_fpga_realtime.rs:452]: In a future PR, we should probably consider exposing an actual cycle count register from the FPGA testbench RTL.

File: hw-model/src/output.rs (hunk 2 above)
Comment (jlmahowa-amd) [reply hw-model/src/output.rs]: Removed

File: hw-model/src/model_fpga_realtime.rs (hunk 3 above)
Comment (jlmahowa-amd) [reply hw-model/src/model_fpga_realtime.rs:452]: I agree. I don't love the assumption about the FPGA speed I had to make here.
Should be easy to do.

**Review by korran — APPROVED** [review]

**Pull request merged**